services/web/server/src/simcore_service_webserver/groups/schemas.py

```diff
@@ -154,3 +155,9 @@ class Config:
                 "delete": False,
             }
         }
+
+
+# Avoids pycln to remove import
+assert convert_groups_db_to_schema  # nosec
+
+__all__: tuple[str, ...] = ("convert_groups_db_to_schema",)
```

Comment:
I don't understand why doing this?
Can't the `convert_groups_db_to_schema` be imported from the module it is defined in?
Is this to be used outside of the package?

Reply:
as discussed!